BrentOzarULTD/SQL-Server-First-Responder-Kit
# Add priority to the output of sp_BlitzLock

Put it right next to log_used, since these go together to determine victim choice.

Fixes #1568.

Changes proposed in this pull request:
 - Add priority to the output of the first (detailed) resultset

How to test this code:
 - Run `exec sp_BlitzLock` on a machine that has deadlocks to be analyzed
 - Verify the "priority" column is there between 

Has been tested on (remove any that don't apply):
 - ~~Case-sensitive SQL Server instance~~
 - SQL Server 2012
 - ~~SQL Server 2014~~
 - SQL Server 2016
 - ~~SQL Server 2017~~
 - ~~Amazon RDS~~
 - ~~Azure SQL DB~~

Note about testing: since this was just accessing one more property in the same XML document being read throughout the script, I think the risk is low that I introduced version-specific errors, so I only tested on versions that I have installed already
